csi/moac/pvhandler.ts

```diff
@@ -0,0 +1,74 @@
+const log = require('./logger').Logger('pvhandler');
```

Comment:
Can we add tests for this?

Reply:
Given that there is are e2e tests, and the code paths in this change are exercised every time a volume is deleted, I see no merit in creating mock tests.